.eslintrc

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "react-app"
+    "extends": "react-app",
```

Comment:
Ah, that makes sense. Is there a way to get more output from stickler?
What presets do you support (if any)? I was unsure if I was getting the .stickler.yml wrong, or something else